frontend/src/components/Pages/InspectionPage/InspectionSection.tsx

```diff
@@ -67,22 +68,23 @@ export interface OngoingMissionType {
 export const compareInspections = (i1: Inspection, i2: Inspection) => {
     if (!i1.missionDefinition.inspectionFrequency) return 1
     if (!i2.missionDefinition.inspectionFrequency) return -1
-    if (!i1.missionDefinition.lastRun) return 1
-    if (!i2.missionDefinition.lastRun) return -1
+    if (!i1.missionDefinition.lastSuccessfulRun) return 1
+    if (!i2.missionDefinition.lastSuccessfulRun) return -1
```

Comment:
Is it best practise for typescript to use -1 and 1 rather than none or bools?

Reply:
This is the expected output of the function https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#parameters